AP_Randomizer/apworld/rules_normal.py

```diff
@@ -184,6 +237,66 @@ def __init__(self, world) -> None:
         }
 
         location_clauses = {
+            "Empty Bailey - Solar Wind": lambda state:
+                self.has_slide(state),  # to consider: damage boosting w/ crouch
+            "Empty Bailey - Cheese Bell": lambda state:  # TODO consider to/from center steeple
+                self.can_slidejump(state)  # untested
+                and (
+                    self.kick_or_plunge(state, 2)
+                    or self.has_gem(state))
+                or self.get_kicks(state, 3),
+            "Empty Bailey - Inside Building": lambda state:
+                self.has_slide(state),
+            "Empty Bailey - Center Steeple": lambda state:
+                self.has_plunge(state),
+            "Empty Bailey - Guarded Hand": lambda state:
+                upper_bailey.can_reach(state)
+                and (
+                    self.knows_obscure(state)
+                    or self.has_gem(state)
+                    or self.get_kicks(state, 3))
+                or self.has_breaker(state)   # do the fight
+                and (
+                    self.has_plunge(state)
+                    or self.get_kicks(state, 2)),
+            "Twilight Theatre - Soul Cutter": lambda state:
+                self.can_strikebreak(state)
+                and (  # we probably already have some of this movement but worth marking it imo
+                    self.can_bounce(state)
+                    or self.kick_or_plunge(state, 1)
+                    or self.has_gem(state)
+                    or self.can_slidejump(state)),
+            "Twilight Theatre - Corner Beam": lambda state:
+                self.has_gem(state)
+                and (
+                    self.has_plunge(state)
+                    or self.get_kicks(state, 2)
+                    or self.can_slidejump(state))  # untested
+                or self.has_plunge(state) and self.get_kicks(state, 3) and self.knows_obscure(state)  # use crouch backflip
+                or self.get_kicks(state, 4),
+            "Twilight Theatre - Locked Door": lambda state:
+                self.has_small_keys(state)
+                and (
+                    self.can_bounce(state)
+                    or self.get_kicks(state, 3)
```

Comment:
1 kick is pretty easy, get on top of the door and kick along the wall. if not normal, you could add it to hard but it's not in any rule

Reply:
felt a lot harder when i tried earlier, yea 1 kick's fine
```suggestion
                    or self.get_kicks(state, 1)
```